backend/src/main/java/com/stackunderflow/backend/controller/CommentController.java

```diff
@@ -1,32 +1,77 @@
 package com.stackunderflow.backend.controller;
 
+import com.stackunderflow.backend.DTOS.CommentDTO;
+import com.stackunderflow.backend.DTOS.EditAnswerDTO;
+import com.stackunderflow.backend.DTOS.EditCommentDTO;
+import com.stackunderflow.backend.DTOS.Message;
 import com.stackunderflow.backend.DTOS.SaveCommentDTO;
+import com.stackunderflow.backend.DTOS.SuggestionDTOAns;
 import com.stackunderflow.backend.model.Comment;
 import com.stackunderflow.backend.service.CommentService;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.security.Principal;
 import java.util.List;
 
 @Slf4j
 @RestController
-@RequestMapping("/api/comment")
+@RequestMapping("/api/answer")
 @RequiredArgsConstructor
 public class CommentController {
     private final CommentService commentService;
 
-    @PostMapping("/save")
-    void saveComment(@RequestBody SaveCommentDTO comment){
-        commentService.saveComment(comment);
+    @PostMapping
+    Message saveComment(@RequestBody SaveCommentDTO comment, Principal principal){
+        return commentService.saveComment(comment, principal.getName());
     }
 
-    @GetMapping("/all")
+    @GetMapping("/allComments")
     List<Comment> getAllComments(){
         return commentService.getAllComments();
     }
+
+    @GetMapping("/{id}")
+    CommentDTO getCommentById(@PathVariable Long id){
+        return commentService.getCommentById(id);
+    }
+
+    @PostMapping("/edit/{id}")
```

Comment:
Put / Patch

Reply:
Am pus post pt ca se adauga in tabelul de sugestii